server/graphql/v2/input/OrderCreateInput.ts

```diff
@@ -139,7 +139,66 @@ export const PendingOrderCreateInput = new GraphQLInputObjectType({
     },
     toAccount: {
       type: new GraphQLNonNull(AccountReferenceInput),
-      description: 'The collective/event/project you want to contribute to',
+      description: 'The collective you want to contribute to',
+    },
+    childAccount: {
+      type: AccountReferenceInput,
+      description: 'The event/project you want to contribute to',
+    },
```

Comment:
I don't get why we need to pass both `toAccount` and `childAccount`, the parent can always be retrieved from `child.getParentCollective()` right?

Reply:
We don't have to pass both. In fact, you can see that we're using `childAccount` as the `toAccount` if it is passed.
The thing is, since in the frontend it is optional to pick an event/project for the collective, it is harder to manage the state of both values as being `toAccount` in the frontend.

My cheap solution was to just send both as independent values and decided which one to use in the backend.

Reply:
Maybe we can keep storing two values in the frontend, then handle that during the `submit`?

```es6
submit({ variables: { toAccount: childAccount || toAccount })
```

I understand the workaround, but I don't think introducing this complexity in the API is the right thing to do.

Reply:
You're right. I think I ended up overcomplicating all of this. It seems pretty obvious now.